apps/api/src/domain/games/memory-match/models/memory-match-round.spec.ts

```diff
@@ -448,4 +448,16 @@ describe(`MemoryMatchRound`, () => {
             });
         });
     });
+
+    describe(`remove`, () => {
+        describe(`when the update is valid`, () => {
+            it.todo(`should remove the round`);
+        });
+
+        describe(`when the update is invalid`, () => {
+            describe(`when the round is already removed`, () => {
```

Comment:
It's not a round we're removing, but a card. I'd probably word this case as "when there is no card with the given sequence number". In that case, you can build a memory match round with `{cards: [targetCard]}` and then call `const result = testRound.remove(999)` and make sure you get an appropriate error. Word the error `You cannot remove card ${sequenceNumber} as there is no such card in match round ${round.id}`.